maintainers/maintainer-list.nix

```diff
@@ -5951,6 +5951,12 @@
     githubId = 11947756;
     name = "Julien Dehos";
   };
+  julienmalka = {
+    email = "julien.malka@me.com";
+    github = "JulienMalka";
+    githubId = 1792886;
+    name = "Julien Malka";
+  };
```

Comment:
Could you put this [in a separate commit](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers) with commit message `maintainers: add julienmalka` in this PR, and put the package update change in another single commit with commit message `htpdate: 1.2.2 -> 1.3.1`?